### DISCUSSION TITLE
No version after 2.5.0 works for me

### DISCUSSION THREAD

**Discussion by lbandi2** [timmo001/system-bridge#942] [Help] [answered]
Hi, I've tried every version after 2.5.0 and none of them work on my system (Windows 10).
That means, I can't access the 'Open Settings' option because the webpage times out, same as the 'View Data' option.

I also noticed that every version after 2.6.0 won't properly close when I click the 'Exit' option on the system tray. I have to go to task manager and kill the Node.js processes in order to actually force it to exit.

I would like to attach my log file but I don't know how to do that since this is my first time posting on github so I pasted it on [Pastebin](https://pastebin.com/u1AZCMtH)

Hope any of this helps and I'm also willing to lend a hand whichever way I can with the testing of this software which is awesome and very my appreciated. So if you need me to provide files, info, insight or time to test stuff I'm up for it, just let me know.

**Comment by timmo001** [comment, marked as answer]
A fix has been released for #869. Please update

**Reply by lbandi2** [reply]
Version 2.7.6 solved it for me, thanks for the quick fix/response!
